sklearn/utils/testing.py

```diff
@@ -882,3 +882,128 @@ def check_docstring_parameters(func, doc=None, ignore=None, class_name=None):
+                                             object_name+" has inconsistency.")
+        else:
+            if include is None:
+                if name not in exclude:
+                    add_dict = {}
+                    add_dict['type_definition'] = type_definition
+                    add_dict['description'] = description
+                    type_dict[name] = add_dict
+            else:
+                if include == '*' or name in include:
+                    add_dict = {}
+                    add_dict['type_definition'] = type_definition
+                    add_dict['description'] = description
+                    type_dict[name] = add_dict
+
+    return type_dict
+
+
+def assert_consistent_docs(objects,
+                           include_params=None, exclude_params=None,
+                           include_attribs=None, exclude_attribs=None,
+                           include_returns=None, exclude_returns=None):
+    """
+    Checks consistency between the docstring of ``objects``.
+
+    Checks if types and descriptions of Parameters/Attributes/Returns are
+    identical across ``objects``. Raises AssertionError if found otherwise.
+
+    Parameters
+    ----------
+    objects : list
+        The list of objects that may be either ``NumpyDocString`` instances or
+        objects (classes, functions, descriptors) with docstrings that can be
+        parsed as numpydoc.
+
+    include_params : list, '*' or None (default)
+        List of Parameters to be included. '*' for including all parameters.
+
+    include_attribs : list, '*' or None (default)
+        List of Attributes to be included. '*' for including all attributes.
+
+    include_returns : list, '*' or None (default)
+        List of Returns to be included. '*' for including all returns.
+
+    exclude_params : list, '*' or None (default)
+        List of Parameters to be excluded. '*' for excluding all parameters.
+
+    exclude_attribs : list, '*' or None (default)
+        List of Attributes to be excluded. '*' for excluding all attributes.
+
+    exclude_returns : list, '*' or None (default)
+        List of Returns to be excluded. '*' for excluding all returns.
+
+    Examples
+    --------
+    >>> from sklearn.metrics import (mean_absolute_error, mean_squared_error,
+    ... median_absolute_error)
+    >>> from sklearn.utils.testing import assert_consistent_docs
+    ... # doctest: +SKIP
+    >>> assert_consistent_docs([mean_absolute_error, mean_squared_error],
+    ... include_params=['y_true', 'y_pred', 'sample_weight'],
+    ... exclude_attribs='*', exclude_returns='*')  # doctest: +SKIP
+    >>> assert_consistent_docs([median_absolute_error, mean_squared_error],
+    ... include_params='*', exclude_attribs='*', exclude_returns='*')
+    ... # doctest: +SKIP
+    Traceback (most recent call last):
+        ...
+    AssertionError: Parameter y_true of mean_squared_error has inconsistency.
+
+    """
+    try:
```

Comment:
I added it because the doctests were failing, I will remove them from here and let it be in the tests. Would the helper be better or just raise a skip inside the test?